activerecord/lib/active_record/enum.rb

```diff
@@ -346,6 +346,12 @@ def assert_valid_enum_definition_values(values)
           if values.keys.any?(&:blank?)
             raise ArgumentError, "Enum values #{values} must not contain a blank name."
           end
+
+          if values.values.any?(Symbol)
```

Comment:
I don't think `assert_valid_enum_definition_values(values)` is the right place to put it. this is first implementation to validate the fix

Reply:
I think it's fine. But rather to check just for symbols, it should check the type of all values to ensure it's something excpected.

Reply:
I found that possible value types are `[Integer, String, Symbol, TrueClass, FalseClass, NilClass]`.
Is testing that the value types are in the array above or should I check that it contains only Integer or only boolean or only String and Symbols ?

Reply:
@byroot I updated the code. I hope it does what you meant.